### ISSUE TITLE
Trickle Accessibility 

### ISSUE THREAD

**Issue by lc-thomasberger** [adaptlearning/adapt_framework#1763]
When step-locking is active, tabbing to the end of the Page turns the page blank.

When the bottom of the page label is focused, the page is scrolled down. The blank page is a result of blocks having `visibility-hidden` class applied.
There is also no Screen-reader feedback when step-locking is active that inform users that something is missing to proceed.
